src/background/background-init.ts

```diff
@@ -121,7 +121,7 @@ async function initialize(): Promise<void> {
         logger,
     );
 
-    const chromeCommandHandler = new ChromeCommandHandler(
+    const chromeCommandHandler = new KeyboardShortcutHandler(
```

Comment:
Should we also change the variable name `chromeCommandHandler` to `keyboardShortcutHandler`?

Reply:
Good catch!
Updated.